samples/client/petstore/cpp-qt5/client/CMakeLists.txt

```diff
@@ -14,12 +14,25 @@ endif ()
 find_package(Qt5Core REQUIRED)
 find_package(Qt5Network REQUIRED)
 
-file(GLOB SRCS
-    ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp
+add_library(${PROJECT_NAME}
+    PFXApiResponse.cpp
+    PFXCategory.cpp
+    PFXOrder.cpp
+    PFXPet.cpp
+    PFXTag.cpp
+    PFXUser.cpp
+    PFXPetApi.cpp
+    PFXStoreApi.cpp
+    PFXUserApi.cpp
+    PFXHelpers.cpp
+    PFXHttpRequest.cpp
+    PFXHttpFileElement.cpp
 )
+target_link_libraries(${PROJECT_NAME} PRIVATE Qt5::Core Qt5::Network )
 
-add_library(${PROJECT_NAME} ${SRCS})
-target_link_libraries(${PROJECT_NAME} PRIVATE Qt5::Core Qt5::Network ssl crypto)
+if(UNIX AND NOT APPLE)
```

Comment:
what about windows?

Reply:
I fixed that here: 5367ed8104b2925ec66fa45e5ed7ea850f76a567

Unfortunately I wasn't able to test on Windows but it shouldn't change the behavior now!

Reply:
Let's see if our community (window users) can help us confirm that after merging this PR into master.